frontend/src/routes/ApplicationPortal.tsx

```diff
@@ -69,6 +71,14 @@ const ApplicationPortal = () => {
     initializeState();
   }, []);
 
+  useEffect(() => {
+    // Only run on first load after myApplication is set
+    if (isEditingApplication === null && myApplication !== undefined) {
+      // Set to is not editing if myApplication exists (user has submitted an application)
+      setIsEditingApplication(!myApplication);
+    }
+  }, [isEditingApplication, myApplication]);
```

Comment:
This logic sets what `isEditingApplication` should be on page load (the variable that decides if you see the page where you edit your application OR the receipt page you get after submitting the application).  
The way I set it to now, it will always show the receipt page if you have previously submitted an application.

A more ideal behavior might be to save a timestamp to the application draft (in session storage) and compare that to the timestamp of the submitted application

Reply:
how did this work before?

Reply:
Previously it worked the same way as it does now, although with a little bug where you had to press the submit-button twice when you were making an application (because the initial value was still null, and not true or false).  
So the option to change stuff is more of a future improvement so to say